### PR TITLE
3.17.0

### PR DESCRIPTION
**Pull request by astorm**
Release (post dated to July 5th, 2022)

### CODE REVIEW

**Comment by astorm** [comment]
jenkins run the tests please

**Comment by astorm** [comment]
jenkins run the tests please

**Comment by trentm** [comment]
Test failures above are all spurious AFAICT.

File: CHANGELOG.asciidoc [hunk 1]
```diff
@@ -28,7 +28,7 @@ Notes:
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
 
-==== Unreleased
+==== 3.17.0 2021/07/5
```
Comment (trentm) [inline CHANGELOG.asciidoc:31]:
```suggestion
==== 3.17.0 2021/07/05
```

File: docs/upgrading.asciidoc [hunk 2]
```diff
@@ -31,6 +31,7 @@ The table below is a simplified description of this policy.
 [options="header"]
 |====
 |Agent version |EOL Date |Maintained until
+|3.17.x |2022-01-05 |3.18.0
```
Comment (trentm) [inline docs/upgrading.asciidoc:34]:
```suggestion
|3.17.x |2023-01-05 |3.18.0
```

File: CHANGELOG.asciidoc [hunk 3]
```diff
@@ -28,7 +28,7 @@ Notes:
...
 
 [float]
 ===== Breaking changes
```
Comment (trentm) [inline CHANGELOG.asciidoc:34]:
```suggestion
```

File: CHANGELOG.asciidoc [hunk 4]
```diff
@@ -28,7 +28,7 @@ Notes:
 [[release-notes-3.x]]
 === Node.js Agent version 3.x
```
Comment (trentm) [inline CHANGELOG.asciidoc:30]:
```suggestion
[[release-notes-3.17.0]]
```

**Review by trentm — CHANGES_REQUESTED** [review]

**Review by trentm — APPROVED** [review]

**Comment by trentm** [comment]
v3.17.0 tagged (https://github.com/elastic/apm-agent-nodejs/releases/tag/v3.17.0) and published (https://www.npmjs.com/package/elastic-apm-node/v/3.17.0)

**Pull request merged**